lib/_http_outgoing.js

```diff
@@ -146,11 +148,29 @@ function OutgoingMessage() {
 
   this._keepAliveTimeout = 0;
 
-  this._onPendingData = nop;
```

Comment:
Should we leave this line so the change is not semver-major (even though it seems highly doubtful it would break anyone if it doesn't make any of our tests fail) and it doesn't look directly related to other changes?

Reply:
This is a bad merge. It should not be removed. I will fix.